nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -1145,11 +1145,29 @@ systemd.services.nginx.serviceConfig.User = lib.mkForce "root";
      As well as this, the options <literal>security.acme.acceptTerms</literal> and either
      <literal>security.acme.email</literal> or <literal>security.acme.certs.&lt;name&gt;.email</literal>
      must be set in order to use the ACME module.
-     Certificates will be regenerated on activation, no account or certificate will be migrated from simp-le.
-     In particular private keys will not be preserved. However, the credentials for simp-le are preserved and
-     thus it is possible to roll back to previous versions without breaking certificate generation.
-     Note also that in contrary to simp-le a new private key is recreated at each renewal by default, which can
-     have consequences if you embed your public key in apps.
+    </para>
+    <para>Please note:
+     <itemizedlist>
+      <listitem><para>Certificates will be regenerated on activation, no account or certificate will be migrated from simp-le.
+        In particular private keys will not be preserved. However, the credentials for simp-le are preserved and
+        thus it is possible to roll back to previous versions without breaking certificate generation.
+        Note also that in contrary to simp-le a new private key is recreated at each renewal by default, which can
+        have consequences if you embed your public key in apps.
+       </para>
+       <para>
+        A separate Let's-Encyrpt-account is registered for each distinctive
+        <literal>security.acme.email</literal>/<literal>security.acme.certs.&lt;name&gt;.email</literal>.
+        If you have several certificates, this may trigger Let's-Encrypt account rate limits during the update
+        (currently max. 10 accounts each 3h). To prevent this, you can use the same email-address for all your
+        certificates.
+       </para>
+      </listitem>
+      <listitem><para>By default, now "ec384"-certificates are created instead of rsa-certificates.
+        You may want to change this by setting <literal>security.acme.certs.&lt;name&gt;.keyType</literal>
+        back to e.g. <literal>rsa4096</literal>, especially if you are using the certificate for a mailserver.
```

Comment:
Can you elaborate a bit on this? Why do my mailservers need different certificates?

Reply:
For interoperatbility, mailservers need RSA certificates.
There are several mailservers out there, which do not yet support ec384-certificates. Now, if you use ec384-certificates, these mailservers cannot send mail to you and send bounce-messages back to the sender (even if encryption is optional in your mailserver). So, if you do this, you will lose mails, cause bounces and be unsubscribed from mailinglists.

So, for a mailserver, you *must* have an RSA certificate.
See also: http://postfix.1071664.n5.nabble.com/TLS-problem-no-shared-cipher-tp105970p105979.html

Reply:
Thanks to the explanation! We should add a small part of it to the `security.acme.certs.<name>.keyType` option, WDYT?

Reply:
Note that on master (but not 20.03) the default key type is `ec256` instead; see https://github.com/NixOS/nixpkgs/pull/83121#issuecomment-630260135.

Reply:
We backported the change to use `ec256` to 20.03 as well, as that's what `lego` will default to in their next version: https://github.com/go-acme/lego/issues/1091